WordPress/gutenberg
# [RNMobile]: iOS DarkMode Improvements

## Description
This PR removes some declarations not needed for iOS DarkMode from RichText.
This helps to fix https://github.com/wordpress-mobile/gutenberg-mobile/issues/1391

## How has this been tested?
- Checkout `gutenberg-mobile` PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1415
- Run the project on iOS 13 with dark mode.
- Add a List block.
- Check that the placeholder is properly lied out.
